.github/workflows/pull_request.yml

```
@@ -1,38 +1,25 @@
-# Copyright 2020 The OpenEBS Authors. All rights reserved.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-name: ci
+name: Pull Request CI
 
 on:
   pull_request:
     paths-ignore:
-      - 'deploy/helm/**'
       - 'docs/**'
       - 'design/**'
       - 'changelogs/**'
       - '*.md'
+      - 'MAINTAINERS'
     branches:
-      # on pull requests to develop and release branches
       - develop
-      - 'v*'
+      - release/*
```

This filter is invalid without the single quotes.

nit: should we narrow the filter down? [Look here](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet).
```suggestion
      - 'release/v[0-9]+.[0-9]+'
```

---

@niladrih Don't think we need the v here right? The names would be release/10.11 and not release/v10.11 ?

---

Yeah, the v should not be there.